pkgs/games/rrootage/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, fetchpatch, fetchurl, SDL, SDL_mixer, bulletml }:
+
+let
+  version = "0.23a";
+  debianRevision = "12";
+  debianPatch = patchname: hash: fetchpatch {
+    name = "${patchname}.patch";
+    url = "https://sources.debian.org/data/main/r/rrootage/${version}-${debianRevision}/debian/patches/${patchname}.patch";
+    sha256 = hash;
+  };
+
+in stdenv.mkDerivation {
+  name = "rrootage-${version}";
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/rrootage/rRootage-${version}.tar.gz";
+    sha256 = "01zzg4ih3kmbhsn1p9zr7g8srv1d2dhrp8cdd86y9qq233idnkln";
+  };
+
+  patches = [
```

Comment:
Same here.

Reply:
I'm checking right now, but I'm unsure about some.

For example removing [this patch](https://sources.debian.org/src/rrootage/0.23a-12/debian/patches/10_deg_out_of_range.patch/) seems to have no effect on the runtime behavior of rrootage, but looking at the description it makes sense to include it anyway.

What do you think I should do?

Reply:
Or the "high res by default" patch or the "windowed mode" patch, which are both _really_ useful but not strictly necessary

Reply:
@Mic92 ping?

Maybe I should just fork that repository and apply those patches...